.github/workflows/cypress-run.yml

```diff
@@ -91,5 +91,5 @@ jobs:
       - uses: actions/upload-artifact@v3
         if: ${{ failure() }}
         with:
-          name: logs-${{ matrix.project }}
-          path: /home/runner/.vegacapsule/testnet/logs
+          name: test-report-${{ matrix.project }}
+          path: frontend-monorepo/apps/trading-e2e/cypress/reports/html
```

Comment:
The upload artifact is currently used for capsule logs by all apps. I think the name and path is fine as it was before?

Reply:
I revered logs